OverlayPlugin.Core/MemoryProcessors/Combatant/CombatantMemory63.cs

```diff
@@ -101,6 +102,17 @@ protected override unsafe Combatant GetCombatantFromByteArray(byte[] source, uin
                 combatant.IsTargetable =
                     (combatant.ModelStatus == ModelStatus.Visible)
                     && ((combatant.Status == ObjectStatus.NormalActorStatus) || (combatant.Status == ObjectStatus.NormalSubActorStatus));
+
+                var drawObjectBytes = memory.GetByteArray(mem.DrawObjectAddr, DrawObject.Size);
+                if (drawObjectBytes?.Length > 0)
+                {
+                    fixed (byte* dptr = &drawObjectBytes[0])
+                    {
+                        IntPtr intPtr = new IntPtr((void*)dptr);
+                        var drawObj = (DrawObject)Marshal.PtrToStructure(intPtr, typeof(DrawObject));
+                        combatant.IsVisible = mem.DrawObjectAddr != IntPtr.Zero && (drawObj.Flags & 1UL) != 0;
+                    }
+                }
```

Comment:
It probably makes more sense to check for `IntPtr.Zero` before trying to read the memory instead.

Additionally, I believe that this can be simplified a bit further via some direct type casting rather than going through Marshal.PtrToStructure.

Finally, the check for length of `drawObjectBytes` does nothing because the function will always return a byte array of the expected length, regardless of the success of the memory read call.

```suggestion
                if (mem.DrawObjectAddr != IntPtr.Zero) {
                    var drawObjectBytes = memory.GetByteArray(mem.DrawObjectAddr, DrawObject.Size);
                    fixed (byte* dptr = &drawObjectBytes[0])
                    {
                        var drawObj = *(DrawObject*)&dptr[0];
                        combatant.IsVisible = (drawObj.Flags & 1UL) != 0;
                    }
                }
```

I might be missing something else here, so I'll wait for @quisquous or someone else to weigh in to make sure.